README.md

```diff
@@ -24,6 +24,13 @@ For convenience, you can alternatively run the `make.sh` script.
 - CMake for generating the build files
 - Python 3 for the [code generator](https://github.com/toru/h2olog/blob/v2/misc/gen-quic-bpf.py)
 - [BCC](https://iovisor.github.io/bcc/) (>= 0.11.0) [installed](https://github.com/iovisor/bcc/blob/master/INSTALL.md) on your system
+- systemtap for Linux to enable DTrace for H2O
```

Comment:
Thanks, this is an important one. I think this explanation is a bit far-fetched. systemtap provides a DTrace shim to produce header files and stubs. How about literally: "systemtap for the DTrace shim script"?

Reply:
bbab467

Yeah, it is important but a bit complicated.

Reply:
On second thought, isn't this an H2O requirement?

Reply:
After sleeping on it, I think this content is better suited for: https://github.com/h2o/h2o/blob/master/doc/install.html

It's kind of implied in the runtime requirement (section below) that developers need to figure this out when building H2O with `-DWITH_DTRACE=on` option. Therefore I think this instruction is better suited for H2O's documentation. Once it's in H2O's documentation, perhaps we could link to it from the bullet point.

Reply:
Agreed. It seems to be better. Sent a pull-req to h2o to explain it: https://github.com/h2o/h2o/pull/2319

and removed it from here.

Reply:
Awesome! thank you (for the work in both repos).